src/Cache/Cache.php

```diff
@@ -463,6 +489,18 @@ public static function clear(string $config = 'default'): bool
         return static::pool($config)->clear();
     }
 
+    /**
+     * Delete all keys from the cache asynchronously.
+     * But, it is not a fully asynchronous method because keys selection portion is synchronous.
```

Comment:
Should this comment mention that this approach is 'blocking' not asynchronous?

Reply:
Sorry, this PR was still a DRAFT and I will mention it to you when it is READY.